src/systemctl/systemctl.c

```diff
@@ -273,6 +275,7 @@ static int systemctl_help(void) {
                "                         sleeping, or hibernating\n"
                "  -i                     Shortcut for --check-inhibitors=no\n"
                "     --kill-whom=WHOM    Whom to send signal to\n"
+               "  -v --kill-value=INT    Signal value to enqueue\n"
```

Comment:
bikeshedding: `-v` is going to cause surprises here, as normally you'd expect to either be a shortcut for `--version` or `--verbose`. Is it really needed to add a shortcut for it, given `--kill-whom` doesn't have one?